crates/graphql/src/schema/objects/nft.rs

```diff
@@ -290,9 +292,58 @@ impl NftActivity {
     }
 }
 
-#[derive(Debug, Clone)]
+#[graphql_interface(for = [BaseNft, CollectionNft])]
+#[async_trait]
+pub trait Nft {
```

Comment:
NftExt of AsNft or BaseNft

I'd like to keep Nft holding its name its less confusing to developers. I was researching some trait naming conventions thinking NftExt is the most appropriate here.
https://internals.rust-lang.org/t/naming-convention-for-traits/1796/9
https://github.com/rust-lang/api-guidelines/discussions/28#discussioncomment-233418

Reply:
I've been trying to figure out how to implement this in a way that does not break Holaplex.com and marketplaces. So far this implementation is the closest, yet still falls short because of the issue I posted in slack. Renaming the trait to NftExt will also lead to breaks.